plugins/ndr/parser.cpp

```diff
@@ -175,10 +177,16 @@ void _ReadShaderAttribute(const UsdAttribute &attr, NdrPropertyUniquePtrVec &pro
 
         hints.insert({TfToken(it.first), TfStringify(it.second)});
     }
+    // We're explicitely using token types for closures, to be consistent with other shader libraries.
+    // But the declared type must be "Terminal"
+    if (typeName == SdfValueTypeNames->Token) {
```

Comment:
what are the other shader libraries ? I am wondering if we shouldn't use "opaque" instead as token is for "interned string with fast comparison and hashing" which we might want to use for shaders ... what do you think ?
https://openusd.org/release/api/_usd__page__datatypes.html

Reply:
just seeing https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/usd/usdMtlx/utils.cpp#L539 
so I understand why now